onnx/defs/math/defs.cc

```diff
@@ -2594,4 +2594,157 @@ ONNX_OPERATOR_SET_SCHEMA(
             propagateShapeFromInputToOutput(ctx, 0, 1);
           }
         }));
+
+static int64_t fft_signal_ndim_default = 1;
+
+static const char* FFT_ver13_doc = R"DOC(
+Fast Fourier Transform.
+
+This implementation suppose the DFT (Discrete Fourier Transform) defined by
+```
+A_k =  \sum_{m=0}^{n-1} a_m \exp\left\{-2\pi i{mk \over n}\right\}
+\qquad k = 0,\ldots,n-1.
+```
+)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    FFT,
```

Comment:
The numpy version appears to have some extra attributes: output-length adjustment (relative to input-length), and a normalization mode. They also support real/complex inputs. We should also specify how the output is laid out (which index corresponds to positive frequency and which for negative frequency, etc.)